homeassistant/components/habitica/strings.json

```diff
@@ -4,18 +4,32 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
     },
     "error": {
-      "invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "step": {
       "user": {
+        "menu_options": {
+          "login": "Login to Habitica",
+          "advanced": "Login to other instances"
+        },
+        "description": "Connect your Habitica profile to allow monitoring of your user's profile and tasks."
+      },
+      "login": {
+        "data": {
+          "username": "Email or username (case-sensitive)",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
+      "advanced": {
         "data": {
           "url": "[%key:common::config_flow::data::url%]",
-          "name": "Override for Habitica’s username. Will be used for service calls",
-          "api_user": "Habitica’s API user ID",
-          "api_key": "[%key:common::config_flow::data::api_key%]"
+          "api_user": "User ID",
+          "api_key": "API Token",
```

Comment:
What does habitica use to name their API token/key?

Reply:
they name it `User ID` and  `API Token`